KaterynaKasapchuk/HW7/task3.py

```diff
@@ -0,0 +1,13 @@
+def calculate_characters(word):
```

Comment:
good job
look at this version of code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result